cartographer_ros/cartographer_ros/node.cc

```diff
@@ -179,7 +179,7 @@ void Node::Initialize() {
       &Node::PublishSubmapList, this));
   wall_timers_.push_back(node_handle_.createWallTimer(
       ::ros::WallDuration(options_.pose_publish_period_sec),
```

Comment:
Let's leave this one alone for now until the API settles. We publish lots of things in addition to pose using this period.